src/components/highlight-component/highlight-component.css

```diff
@@ -4,7 +4,7 @@
   position: relative;
 }
 
-.highlight-component::before {
+.highlight-component::after {
```

Comment:
There's probably a lot of restructuring we might want to do to this Sass and most of it is outside the scope of this PR, but I remember having a conversation a while back about `::` versus `:` for pseudo elements, and I think `:` won. Is it worth fixing this here? I think it made it more widely browser compatible.